Tools/ardupilotwaf/boards.py

```diff
@@ -43,6 +43,29 @@ def configure(self, cfg):
         env = waflib.ConfigSet.ConfigSet()
         self.configure_env(cfg, env)
 
+        # Setup scripting, had to defer this to allow checking board size
+        if (not cfg.options.disable_scripting) and ((cfg.env.BOARD_FLASH_SIZE == None) or (cfg.env.BOARD_FLASH_SIZE == []) or (cfg.env.BOARD_FLASH_SIZE > 1024)):
```

Comment:
```suggestion
        if (not cfg.options.disable_scripting) and ((cfg.env.BOARD_FLASH_SIZE is None) or (cfg.env.BOARD_FLASH_SIZE == []) or (cfg.env.BOARD_FLASH_SIZE > 1024)):
```

Break this line.

Reply:
Not sure I'm happy with this being OK with there being no `BOARD_FLASH_SIZE`.  If that should only happen on some platforms I'd rather that be checked explicitly.

Reply:
bloody python.

We don't define it anywhere other then ChibiOS, and I was really trying to avoid having to add methods to each of the overloads...